__tests__/unit/specs/blockchain.spec.js

```diff
@@ -6,12 +6,12 @@ describe('Blockchain Service', () => {
     store.dispatch('network/setServer', 'https://explorer.ark.io/api/v2')
   })
 
-  it('should return the block height', async () => {
+  it('Should return the block height', async () => {
```

Comment:
Why was the casing of this changed? All our projects follow the full lowercase `it('should`.

Reply:
My bad, it was a mix of lower- and uppercase all around.